### DISCUSSION TITLE
Http not working over loopback

### DISCUSSION THREAD

**Discussion by Golempl** [akvorado/akvorado#1286] [Q&A]
Using version 1.11.0 + lastest Zookeeper/kafka/Clickhouse 
All without Docker.

Here are the settings specified in config.yaml:

http:
  listen: 127.0.0.1:8080
  profiler: true
  cache:
    type: memory

The Akvorado Orchestrator is now listening via HTTP on 127.0.0.1:8080. However, an issue arises with ClickHouse:

..
024.06.29 22:53:07.926330 [ 31200 ] {} <Trace> ReadWriteBufferFromHTTP: Failed to make request to 'http://192.168.1.100:8080/api/v0/orchestrator/clickhouse/asns.csv'. Error: Connection refused. Failed at try 5/10. Will retry with current backoff wait is 800/10000 ms.
...

I have not configured 192.168.1.100 anywhere in the settings.

Changing the configuration to:

http:
  listen: :8080
  profiler: true
  cache:
    type: memory

Results in the Orchestrator listening on tcp6 :::8080,
 which allows it to function over the normal network address. 

However, I prefer to restrict this port from external access. 
Adding:
iptables -A INPUT -i ens18 -p tcp --dport 8080 -j DROP   (providing interface is mandatory)
achieves this goal.
Nonetheless, I am curious why the functionality cannot be maintained exclusively over 127.0.0.1.

**Comment by vincentbernat** [comment]
You can override the URL with `orchestrator-url` setting. See https://demo.akvorado.net/docs/configuration#clickhouse.

**Comment by Golempl** [comment]
Thanks, works like a charm.
